gsoc/views.py

```diff
@@ -0,0 +1,31 @@
+from django.shortcuts import render
+
+from .data import get_org_data
+
+
+# Org info view
+def index(request):
+    # We will always get a file here 
+    # (because we also save the fact there is no data for a org)
+    org = get_org_data()
+    for organisation in org.items():
+        if (organisation[0] == 0):
+            # No Data Found
+            name = organisation[1].get('name')
+            return render(request, 'gsoc.html', {'error': name})
+
+        id = organisation[1].get('id')
```

Comment:
why index of `1`?  that is a bit odd.  the yaml should only have one org in it.

Reply:
I got this running this way only. I think that's how it inteprets yaml. Although i will reconsider it and try it via key, value type object. Or it would be good if u can suggest something :).

Reply:
you dont need an array for one org.  the Google json data structure that you fetch should *not* be the same as the coala org yaml data structure that you write to disk.